### PR TITLE
Added tooltip to show English translation of language names

### PR DESCRIPTION
**Pull request by grant-dennison**
Added "englishName" field to language class in parseComCode\main.js

Updated parseComCode\ReadMeParseComCloudCode.txt. I included a more specific link for the CLI tool, added a note on where to run the parse deploy command, and made some notes on necessary manual changes to the global.json file.

### CODE REVIEW

File: parseComCode/ReadMeParseComCloudCode.txt [hunk 1]
```diff
@@ -11,10 +11,36 @@ ParseComTools\parse new
 This asks for an email, which is parse@bloomlibrary.org, and a password, which you will need to know!
 After you enter the right password, it asks which app you want to make cloud code for. Pick an appropriate number from the list.
 This creates a folder under whatever was the current directory when you did it...for example mine is at the root of I:.
+
 The important file (apart from some keys you should protect) is parse/cloud/main.js.
 Replace that file with the version checked in here.
 Edit it however you need to.
+
+The CLI produces a file parse/config/global.json. This file appears to be incompatible with the deploy function.
```
Comment (JohnThomson) [inline parseComCode/ReadMeParseComCloudCode.txt]: CLI = command language interpreter? Do you mean this happens when you run parse new, or every time you deploy, or...? Have you found any indication of whether this is a temporary bug or (for some reason) a new requirement? Might be helpful to say a little more about when and how often you have to make this manual change.

**Comment by grant-dennison** [comment]
CLI = command line interface

I wrote the README chronologically. Thus, there is no instruction to the user of something to do; the implication is that the global.json is created after parse new.

According to this discussion, https://groups.google.com/forum/#!searchin/parse-developers/parse$20windows$20cli$202.0.9/parse-developers/WoRnDft4qmE/kkJeCkYhdAAJ, there is an issue that has developed recently to cause the broken JSON. Currently, the newest version of Parse CLI is broken. When Andrew and I were getting started, we had trouble because the link provided in the README directed us to the broken version. Andrew found the older version that was stable and corresponded to the directions in the README, but the JSON was broken in that version due to recent changes in the Parse.com system. So, I believe the modifications to the README will work for now, but they should be changed when there is a new stable release.

**Comment by grant-dennison** [comment]
One problem other than the JSON even in the older stable version is that it might not be easy to select projects. Changing the JSON manually can also solve this problem however.

**Comment by JohnThomson** [comment]
Grant, it's very helpful to have this information, but I'd prefer to see it in the comments. It's important for the next person to work on this to know that you used an obsolete version of the Parse CLI and that the JSON edits are a hopefully temporary workaround to keep that old version going while the new one is broken. That link would be good to include, as well as where you got the CLI that works (and if it's not already there, where to get the latest).

**Comment by JohnThomson** [comment]
This is getting messy...at least two independent changes. Please make a temporary branch for each distinct change.

**Pull request merged**